src/sass/main.scss

```diff
@@ -2,3 +2,64 @@
 
 @import 'base/base';
 @import 'base/utilities';
+
```

Comment:
Could you move this stuff to a file at `src/sass/layout/_header.scss` please? The main.scss file's purpose is only to import other sass files.